sdk/storage/Azure.Storage.Blobs.Batch/README.md

```diff
@@ -154,7 +154,7 @@ try
     batch.DeleteBlobs(new Uri[] { valid.Uri, invalid.Uri });
 }
 
-catch (AggregateException ex)
+catch (AggregateException)
```

Comment:
Nit - leave the `ex` cause you talk about it two lines down.

Reply:
This causes a compiler error that would need to be suppressed. Suppressing it would pull the suppression comment into the README.